packages/docusaurus-theme-classic/src/theme/Tabs/index.tsx

```diff
@@ -122,7 +123,9 @@ function TabContent({
       // fail-safe or fail-fast? not sure what's best here
       return null;
     }
-    return cloneElement(selectedTabItem, {className: 'margin-top--md'});
+    return cloneElement(selectedTabItem, {
+      className: clsx('margin-top--md', className),
+    });
```

Comment:
Can you prove me that it works thanks to a visible example in the preview?

I suspected the code to not work and it doesn't look like it works

To me you should do `selectedTabItem.props.className` instead